decide/authentication/templates/registration/password_reset_subject.txt

```diff
@@ -0,0 +1,2 @@
+
+Password reset on Decide
```

Comment:
It is necessary so that it does not show the localhost as an issue, it could be removed if we did not launch from local.
It is the subject of the mail that is sent.